application.py

```diff
@@ -14,7 +14,7 @@
 
 def configure_logging():
     log_level = logging.INFO
-    debug = os.getenv("FLASK_ENV") == "development"
+    debug = os.getenv("FLASK_DEBUG")
```

Comment:
Need to ensure this doesn't evaluate `"0"` as debug, currently it would.

Reply:
Updated `debug = os.getenv("FLASK_DEBUG")== "1"